guide_to_writing_techreports.tex

```diff
@@ -1170,21 +1179,22 @@ \section{Appendices}
 journal for a paper with a decent list of references and use it as
 your model.
 
-It is recommended that you use an automated tool to generate your list of references.
+It is recommended that you use an automated tool to generate your list of references, \textit{eg} EndNote, Zotero or Mendeley.
```

Comment:
I feel like modern usage has moved away from italics for common phrases like et al, etc and eg. I’ve tried to replace all my usages of eg with “for example” in an attempt to get rid of many such cases.